README.md

```diff
@@ -5,40 +5,191 @@ Please edit this README.md file with your answers to the worksheet questions.
+|false|true |false|true |true         |true          |
+|false|true |true |false|true         |true          |
+|false|true |true |true |true         |true          |
+|true |false|false|false|true         |true          |
+|true |false|false|true |true         |true          |
+|true |false|true |false|true         |true          |
+|true |false|true |true |true         |true          |
+|true |true |false|false|false        |false         |
+|true |true |false|true |false        |false         |
+|true |true |true |false|false        |false         |
+|true |true |true |true |false        |false         |
 
 ### c
+|A    |B    |C    |D    |(A AND B) OR (A AND C)|A AND (B OR C)|
+|-----|-----|-----|-----|----------------------|--------------|
+|false|false|false|false|false                 |false         |
+|false|false|false|true |false                 |false         |
+|false|false|true |false|false                 |false         |
+|false|false|true |true |false                 |false         |
+|false|true |false|false|false                 |false         |
+|false|true |false|true |false                 |false         |
+|false|true |true |false|false                 |false         |
+|false|true |true |true |false                 |false         |
+|true |false|false|false|false                 |false         |
+|true |false|false|true |false                 |false         |
+|true |false|true |false|true                  |true          |
+|true |false|true |true |true                  |true          |
+|true |true |false|false|true                  |true          |
+|true |true |false|true |true                  |true          |
+|true |true |true |false|true                  |true          |
+|true |true |true |true |true                  |true          |
 
 ### d
+|A    |B    |C    |D    |(A OR B) AND (A OR C)|A OR (B AND C)|
+|-----|-----|-----|-----|---------------------|--------------|
+|false|false|false|false|false                |false         |
+|false|false|false|true |false                |false         |
+|false|false|true |false|false                |false         |
+|false|false|true |true |false                |false         |
+|false|true |false|false|false                |false         |
+|false|true |false|true |false                |false         |
+|false|true |true |false|true                 |true          |
+|false|true |true |true |true                 |true          |
+|true |false|false|false|true                 |true          |
+|true |false|false|true |true                 |true          |
+|true |false|true |false|true                 |true          |
+|true |false|true |true |true                 |true          |
+|true |true |false|false|true                 |true          |
+|true |true |false|true |true                 |true          |
+|true |true |true |false|true                 |true          |
+|true |true |true |true |true                 |true          |
```

Comment:
All correct -- but you should remove the columns for the C and D variables when they are not used (which will also cut down the number of rows in each table)